src/routes/_error.svelte

```diff
@@ -1,30 +1,71 @@
 <script>
+  import MetaData from '../components/UI/MetaData.svelte'
+  import FixedHeader from '../components/UI/Layout/FixedHeader.svelte'
+  import Hotjar from '../components/UI/Tracking/Hotjar.svelte'
+  import GoogleAnalytics from '../components/UI/Tracking/GoogleAnalytics.svelte'
   import { isDevelopment } from '../Consts'
 
   export let status
   export let error
 
   const dev = isDevelopment
+  const title = `Oeps, that's a ${status}`
+  const message = status === 404 ? 'Looks like the page you where looking for is no longer here.' : error.message
 </script>
 
 <style lang="sass">
   @import './assets/style/theme'
 
+  @keyframes rotate
+    0%
+      transform: rotate(0)
+      transform-origin: bottom left
```

Comment:
maybe you could add `transform-origin: bottom left` to the `.logo-svg` and remove it from each step of the animation?

Reply:
Good catch! Done ✔️